src/transformers/models/whisper/modeling_whisper.py

```diff
@@ -1604,6 +1605,8 @@ def generate(
                 Whether to return token-level timestamps with the text. This can be used with or without the
                 `return_timestamps` option. To get word-level timestamps, use the tokenizer to group the tokens into
                 words.
+            num_frames (`float`, *optional*):
+                The number of audio frames available in this chunk. This is only used generating word-level timestamps.
```

Comment:
```suggestion
            num_frames (`float`, *optional*):
                The number of audio frames available in this chunk. This is only used when generating word-level timestamps.
```
I think this should go in the generation config WDYT? (Since you could generate outside the pipeline, with a predifined number of frames.

Reply:
Yep this can go in the generation config! As we have with `generation_config.alignment_heads` for example

Reply:
Note that the nice thing here is that even if we use the generation config to store this property, it will still be updated by any kwargs we pass to the `.generate` method.

E.g. if we set `generation_config.num_frames=None`, but do:
```python
model.generate(input_features, num_frames=num_frames)
```

Then the generation config will be updated with the value of `num_frames` we pass to `.generate`

=> this way, `num_frames` still behaves like a kwarg, but is actually just updated in the `generation_config`

Reply:
Will add to generation config! 👍